bundle/tests/scorecard/config.yaml

```diff
@@ -45,6 +46,12 @@ stages:
     storage:
       spec:
         mountPath: {}
+  - image: docker.io/andreasgerstmayr/scorecard-test-kuttl:1681909083
```

Comment:
Why is it using a custom image? Could we switch to the upstream one?

Reply:
Same as above (this file is auto-generated from the file above).